src/pytkdocs/loader.py

```diff
@@ -913,3 +957,39 @@ def field_is_inherited(field_name: str, fields_name: str, base_class: type) -> b
             *(getattr(parent_class, fields_name, {}).keys() for parent_class in base_class.__mro__[1:-1]),
         ),
     )
+
+
+def split_attr_name(attr_name: str) -> Tuple[str, Optional[str]]:
+    """
+    Split an attribute name into a first-order attribute name and remainder.
+
+    Args:
+        attr_name: Attribute name (a)
+
+    Returns:
+        Tuple containing:
+            first_order_attr_name: Name of the first order attribute (a)
+            remainder: The remainder (b.c)
+
+    """
+    first_order_attr_name, *remaining = attr_name.split(".", maxsplit=1)
+    remainder = remaining[0] if remaining else None
+    return first_order_attr_name, remainder
+
+
+def get_fields(attr_name: str, *, members: Mapping[str, Any] = None, class_obj=None) -> Dict[str, Any]:
+    if not (bool(members) ^ bool(class_obj)):
+        raise ValueError("Either members or class_obj is required.")
+    first_order_attr_name, remainder = split_attr_name(attr_name)
+    fields = members[first_order_attr_name] if members else dict(vars(class_obj)).get(first_order_attr_name, {})
+    if remainder:
+        fields = attrgetter(remainder)(fields)
+
+    if callable(fields):
+        fields = fields()
```

Comment:
No, it's not used at the moment, we can remove it actually.
It was there, because I used _meta.get_fields previously, but I realized that i should use _meta.fields instead.